scripts/build-config.ts

```diff
@@ -22,6 +22,11 @@ export function findBuildConfig(): string {
   let currentDir = process.cwd();
 
   while (!existsSync(resolve(currentDir, BUILD_CONFIG_FILENAME))) {
+    if (currentDir === '/') {
```

Comment:
Will this work for Windows as well?

Reply:
Sorry for not seeing your reply in time. ^_^

Thank you for reminding me. I missed this point.
In Windows, the forward-slash / is generally not used as a path separator in file paths. Instead, the back-slash \ is used as the path separator.

maybe use below to check whether in root:
```
function isCurrentFolderRoot() {
  const rootFolder = path.parse(process.cwd()).root;
  const currentFolder = path.resolve(process.cwd());
  return currentFolder === rootFolder;
}
```